### PR TITLE
Update and fix generator for Volto 17 - 8.x.x series.

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

**Review by davisagli — APPROVED** [review]

**Comment by davisagli** [comment]
@sneridagh hmm, there are some failing generator tests

File: packages/generator-volto/README.md [hunk 1]
```diff
@@ -11,6 +11,14 @@ npm install -g yo
 npm install -g @plone/generator-volto
 ```
 
+## Compatibility
+
+| Generator version | Volto version |
+|-------------------|---------------|
+| 9.x               | >=18.0.0      |
```
Comment (stevepiercy) [inline packages/generator-volto/README.md:18]: I'd only add known good versions, as who knows what will happen in the future?

```suggestion
| 9.x               | 18.x.x        |
```

File: packages/generator-volto/README.md [hunk 2]
```diff
@@ -11,6 +11,14 @@ npm install -g yo
...
+| 7.x and 8.x       | 17.x.x        |
+| 6.x               | 16.x.x        |
+
 ## Usage
 
 ### Creating a new Volto project using `npm init`
```
Comment (stevepiercy) [inline packages/generator-volto/README.md:24]:
```suggestion
### Create a new Volto project using `npm init`
```

File: packages/generator-volto/README.md [hunk 3]
```diff
@@ -115,11 +124,40 @@ Options:
         --ask-answered   # Show prompts for already configured options               Default: false
         --interactive    # Enable/disable interactive prompt                         Default: true
         --template       # Use github repo template, e.g.: eea/volto-addon-template
+        --outputpath     # Output path
 
 Arguments:
   addonName  # Addon name, e.g.: @plone-collective/volto-custom-block  Type: String  Required: false
 ```
 
+### Enabling an existing add-on as a theme add-on
```
Comment (stevepiercy) [inline packages/generator-volto/README.md:133]:
```suggestion
### Enable an existing add-on as a theme add-on
```

File: packages/generator-volto/README.md [hunk 4]
```diff
@@ -115,11 +124,40 @@ Options:
...
+
+If you want one of your add-ons to be a theme, you can run this template on the top of your add-on.
```
Comment (stevepiercy) [inline packages/generator-volto/README.md:135]:
```suggestion
If you want one of your add-ons to be a theme, you can run this template on top of your add-on.
```

File: packages/generator-volto/README.md [hunk 5]
```diff
@@ -137,13 +175,13 @@ Please note that you have to run a Plone backend as well.
 
 E.g. with docker:
 
-```bash
-docker run -it --rm --name=plone -p 8080:8080 -e SITE=Plone -e ADDONS="plone.volto" -e ZCML="plone.volto.cors" -e PROFILES="plone.volto:default-homepage" plone
+```shell
+docker run -it --rm --name=plone -p 8080:8080 -e SITE=Plone -e PROFILES="plone.volto:default-homepage" plone/plone-backend:6.0.7
```
Comment (stevepiercy) [inline packages/generator-volto/README.md:179]: Latest release is 6.0.8.

```suggestion
docker run -it --rm --name=plone -p 8080:8080 -e SITE=Plone -e PROFILES="plone.volto:default-homepage" plone/plone-backend:6.0.8
```

File: packages/generator-volto/README.md [hunk 6]
```diff
@@ -137,13 +175,13 @@ Please note that you have to run a Plone backend as well.
...
 ```
 
-Consult the Volto docs for further details:
+Consult the Plone frontend Volto docs for further details:
```
Comment (stevepiercy) [inline packages/generator-volto/README.md:182]:
```suggestion
Consult the [Plone frontend Volto documentation](https://6.docs.plone.org/volto/index.html) for further details.
```

File: packages/generator-volto/README.md [hunk 7]
```diff
@@ -137,13 +175,13 @@ Please note that you have to run a Plone backend as well.
...
 
-https://www.npmjs.com/package/@plone/volto
+https://6.docs.plone.org/volto/index.html
```
Comment (stevepiercy) [inline packages/generator-volto/README.md:184]: Merged inline above.

```suggestion
```

File: packages/generator-volto/news/5465.bugfix [hunk 8]
```diff
@@ -0,0 +1 @@
+Fix and update generator for Volto 17 - 8.x.x series @sneridagh
```
Comment (stevepiercy) [inline packages/generator-volto/news/5465.bugfix:1]:
```suggestion
Fixed and updated the generator for Volto 17, the 8.x.x series. @sneridagh
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Minor doc fixes.

File: packages/generator-volto/README.md (hunk 1 above)
Comment (stevepiercy) [reply packages/generator-volto/README.md:18]: Actually, this is getting merged to `17.x.x`, so should this line even be here?

**Review by stevepiercy — COMMENTED** [review]
I did not submit this before merge. It's not required, but it would be good to have a reference in the docs instead of updating this section in every branch going forward.

**Pull request merged**